index.md

```diff
@@ -1,64 +1,63 @@
 ---
 layout: homepage
 ---
-<div class="container">
-  <div class="row" style="margin-bottom: 50px; margin-top: 50px;">
-    <div class="col-md-12 center-block clearfix" style="float: none">
-      <div class="col-md-3">
-        <div class="panel panel-primary">
-          <div class="panel-heading">
-            <img src="{{ site.baseurl }}/images/site/icons/forseti-visibility-icon.svg" class="img-fluid" alt="Forseti Security: Keep track of your environment">
-          </div>
-          <div class="panel-body">
-            <p class="text-primary">Keep track of your environment</p>
-            <p>
-              Takes a snapshot of resources on a recurring cadence, so you always have a history of what was in your cloud.
-            </p>
-            <a class="btn btn-link" href="{{ site.baseurl }}/docs/latest/configure/inventory/index.html">More</a>
-          </div>
-        </div>  
+<div class="promo-rows">
+  <div class="promo-row row">
+    <div class="container">
+      <div class="promo-image col-md-2 col-md-offset-1 col-sm-3">
+        <img src="{{ site.baseurl }}/images/site/icons/forseti-visibility-icon.svg" class="img-responsive" alt="Forseti Security: Keep track of your environment">
       </div>
-      <div class="col-md-3">
-        <div class="panel panel-primary">
-          <div class="panel-heading">
-            <img src="{{ site.baseurl }}/images/site/icons/forseti-detection-icon.svg" class="img-fluid" alt="Forseti Security: Monitor your policies"> 
-          </div>
-          <div class="panel-body">
-            <p class="text-primary">Monitor your policies</p>
-            <p>
-              Scans your GCP resources to ensure that access controls are set as you intended and protects against unsafe changes. 
-            </p>
-            <a class="btn btn-link" href="{{ site.baseurl }}/docs/latest/configure/scanner/index.html">More</a>
-          </div>
-        </div>  
+      <div class="promo-text col-md-6 col-sm-9">
+        <h2 class="text-primary">Keep track of your environment</h2>
+        <p>
```

Comment:
We're mixing HTML markup in markdown files here. Is it possible to have this file be purely markdown?